libraries/grpc-sdk/src/index.ts

```diff
@@ -50,12 +54,37 @@ export default class ConduitGrpcSdk {
       this.name = name;
     }
     this.serverUrl = serverUrl;
-    this._config = new Config(this.name, this.serverUrl);
-    this._admin = new Admin(this.name, this.serverUrl);
-    this._router = new Router(this.name, this.serverUrl);
-    this.initializeModules().then(() => {
-    });
+  }
+
+  async initialize() {
```

Comment:
This is not core/db-specific.
Modules now wait for core to be online and serving before moving on with their own initialization.
That way we can launch modules and prevent them from crashing regardless of which order they're brought up in.

For example we could launch Authentication before Core is capable of serving anyone or even started in the first place.
Imo this would make even more sense post-Database-introspection, where Database could take a while before being initially registered for anyone connecting to a populated db.

Reply:
The while loop part isn't ideal either, but using a core health watch callback there would probably be messy as far as module initialization is concerned.

Might still end having to use a watch anyway to prevent core from getting bombarded by requests though.